src/style-spec/reference/v8.json

```diff
@@ -3345,6 +3355,17 @@
           }
         }
       },
+      "distance": {
+        "doc": "Returns the shortest distance in meters between the evaluated feature and the input geometry. The evaluated feature and input value can be a valid GeoJSON of type `Point`, `MultiPoint`, `LineString`, `MultiLineString`, `Polygon`, `MultiPolygon`, `Feature`, or `FeatureCollection`. Results at zoom level 12 and below may vary in precision due to precision loss in conversions between tile coordinates and `[Longitude, Latitude]` at lower zoom levels.",
```

Comment:
> The evaluated feature and input value can be a valid GeoJSON of type

Does this imply that the evaluated feature cannot be part of a vector tile?

> `Point`, `MultiPoint`, `LineString`, `MultiLineString`, `Polygon`, `MultiPolygon`, `Feature`, or `FeatureCollection`

Include details on what distance is measured among some of the pairs and what happens withe `Multi*` and `FeatureCollection` types?

cc @zmiao

Reply:
Hmm, good point, it shouldn't imply that. I'm open to better suggestions! 

> Include details on what distance is measured among some of the pairs and what happens withe Multi* and FeatureCollection types?

Yeah, good idea.

Reply:
@zmiao for the input data, is what I wrote correct about what’s supported?

Yeah, maybe omitting “evaluated” feature from the description could work. What do you think about Asheem’s suggestion to add more detail on which distance is measured and the behavior for Multi* types? Maybe you could provide the info here and we could evaluate whether it’s worth including

Reply:
> for the input data, is what I `wrote` correct about what’s supported?
Yes, it is correct.

> What do you think about Asheem’s suggestion to add more detail on which distance is measured for different combinations and the behavior for Multi* types? 

I think it is a good idea to explain it, I will provide the details soon.